tasmota/xdrv_05_irremote_full.ino

```diff
@@ -167,7 +167,10 @@ uint64_t reverseBitsInBytes64(uint64_t b) {
 \*********************************************************************************************/
 
 const bool IR_FULL_RCV_SAVE_BUFFER = false;         // false = do not use buffer, true = use buffer for decoding
-const uint32_t IR_TIME_AVOID_DUPLICATE = 50;  // Milliseconds
+
+#ifndef IR_TIME_AVOID_DUPLICATE
+#define IR_TIME_AVOID_DUPLICATE = 50           // Milliseconds
```

Comment:
It is not needed the equal sign in a #define

Just delete the sign to pass the CI tests